js_modules/dagit/packages/ui/src/components/TokenizingField.tsx

```diff
@@ -36,14 +36,21 @@ interface TokenizingFieldProps {
   maxValues?: number;
   onChange: (values: TokenizingFieldValue[]) => void;
   onChangeBeforeCommit?: boolean;
+  addOnBlur?: boolean;
```

Comment:
also `addOnPaste` would  be nice

Reply:
See below - I can't seem to get it to work as expected, I think because we override some of the input/change methods.